go.mod

```diff
@@ -177,7 +178,7 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.2.0+incompatible
-	github.com/Shopify/sarama => github.com/elastic/sarama v1.24.1-elastic.0.20200519143807-cbc80333a91e
+	github.com/Shopify/sarama => github.com/elastic/sarama v1.24.1-elastic
```

Comment:
hm, should we tag/version our fork somehow or include the commit ID when vendoring? e.g. apm-server depends on the beats repo, but doesn't vendor. If the checksum changes in the future because we updated our fork this might trip other project builds, no?

Reply:
To be honest I thought I did -- this was vendored targeting a new branch (`beats-vendored`) that was supposed to be the "real" state of the fork from which we could target the latest commit when updating, and I vendored based on a commit hash (355d120d09706d85bc6d2a84287be7fbd13ef453) not on a particular tag or branch, so I'm surprised to see the old tag reappear when the repo contents seem to have correctly pulled in the branch I wanted.

I will look at this more and work with Noémi on how to manage the fork versioning in a way that gives less misleading metadata.